scenariorun/src/main/openapi/scenariorun.yaml

```diff
@@ -302,6 +322,18 @@ paths:
               examples:
                 OneScenarioRun:
                   $ref: '#/components/examples/OneScenarioRun'
+    delete:
+      operationId: deleteHistoricalScenarioRunsByScenario
```

Comment:
Historical data here is not understandable when one read the API: it is the DELETE verb on /organizations/{organization_id}/workspaces/{workspace_id}/scenarios/{scenario_id}/scenarioruns = delete all scenario runs of a scenario, not only the historical one.

As for historical deletion for organizations, create a new endpoint 
DELETE /organizations/{organization_id}/workspaces/{workspace_id}/scenarios/{scenario_id}/scenarioruns/historicaldata
OR
POST /organizations/{organization_id}/workspaces/{workspace_id}/scenarios/{scenario_id}/scenarioruns/deletehistoricaldata
OK for a beta version, but should be corrected for GA.

Reply:
done